FileShareAdminClient/FileShareApiAdminClient.cs

```diff
@@ -153,13 +180,18 @@ public async Task RollBackBatchAsync(IBatchHandle batchHandle)
             }
         }
 
+        public async Task<IResult<RollBackBatchResponse>> RollBackBatchAsync(IBatchHandle batchHandle, CancellationToken cancellationToken = default)
```

Comment:
This method is not covered by any unit tests?

Reply:
Unit test case was written to cover the newly added method RollBackBatchAsync, but the parameter CancellationToken got missed. It is fixed now.